UndertaleModLib/Models/UndertaleFont.cs

```diff
@@ -95,6 +131,8 @@ public void Serialize(UndertaleWriter writer)
             writer.WriteUndertaleObjectPointer(Texture);
             writer.Write(ScaleX);
             writer.Write(ScaleY);
+            if (writer.undertaleData.GeneralInfo?.Major >= 2 && writer.undertaleData.GeneralInfo?.BytecodeVersion >= 17)
```

Comment:
Are you sure this requires GMS2? I guess just bytecode 17 should do it? I don't think it's even possible to have 17 in GMS1.4 (and if you can, who knows what the behavior will be anyway)
```suggestion
            if (writer.undertaleData.GeneralInfo?.BytecodeVersion >= 17)
```